xapian-letor/configure.ac

```diff
@@ -429,15 +429,6 @@ if test -n "${EATMYDATA-unset}" ; then
   AC_PATH_PROG([EATMYDATA], [eatmydata], [])
 fi
 
-dnl Used by tests/harness/testsuite.cc
-AC_CHECK_FUNCS([sigaction])
-AC_MSG_CHECKING([for sigsetjmp and siglongjmp])
-AC_TRY_COMPILE([#include <setjmp.h>],
-  [sigjmp_buf jb; if (sigsetjmp(jb, 1)) { siglongjmp(jb, 1);  }],
-  AC_DEFINE([HAVE_SIGSETJMP], [1], [Define to 1 if you have the 'sigsetjmp' function])
-  AC_MSG_RESULT([yes]),
-  AC_MSG_RESULT([no]))
-
```

Comment:
Ah, that's annoying. Okay, then you want to add something to the readme saying that your flags for that must match whatever you built -core with. (In time we can change the harness so there's a small set of flags to set no matter what in letor's configure, and so it'll just build & run tests with the default disk-based backend, whatever that is.)